### DISCUSSION TITLE
Amundsen implementation with neo4j - can't reach neo4j ui on localhost:7474

### DISCUSSION THREAD

**Discussion by kc-alva** [amundsen-io/amundsen#2032] [Q&A]
Good day everyone, 
I've been trying to implement Amundsen using neo4j to compare it with atlas but I have not been to use the neo4j UI, can't reach it in port localhost:7474, everything else is working properly, I can also load data into neo4j using the example dataloaders.
Has anyone experience this or knows how to solve it in order to access neo4j UI?

**Comment by VoLKyyyOG** [comment]
You may need to port forward
